packages/lib/models/Setting.ts

```diff
@@ -1020,19 +1020,17 @@ class Setting extends BaseModel {
 			'folders.sortOrder.reverse': { value: false, type: SettingItemType.Bool, storage: SettingStorage.File, isGlobal: true, public: true, label: () => _('Reverse sort order'), appTypes: [AppType.Cli] },
 			trackLocation: { value: true, type: SettingItemType.Bool, section: 'note', storage: SettingStorage.File, isGlobal: true, public: true, label: () => _('Save geo-location with notes') },
 
-			// 2020-10-29: For now disable the beta editor due to
-			// underlying bugs in the TextInput component which we cannot
-			// fix. Also the editor crashes in Android and in some cases in
-			// iOS.
-			// https://discourse.joplinapp.org/t/anyone-using-the-beta-editor-on-ios/11658/9
+			// 2023-09-07: This setting is now used to track the desktop beta editor. It
+			// was used to track the mobile beta editor previously.
 			'editor.beta': {
```

Comment:
The existing `editor.beta` setting seemed to be unused. Here, it is made visible on desktop under the "general" section (where the some other markdown editor settings are).

Would it be better to create a new setting for the desktop beta editor?

Reply:
Using the existing property is fine. It's been used for various editors over the years both for mobile and desktop